package.json

```diff
@@ -13,18 +13,19 @@
     ]
   },
   "scripts": {
-    "test": "set NODE_ENV=test && npm run migration:undo && npm run migration && npm run seeds && nyc --reporter=html --reporter=text mocha ./server/tests/*.test.js --exit --require @babel/register",
+    "test": "set NODE_ENV=test && npm run migration:reset && nyc --reporter=html --reporter=text mocha --require @babel/register ./server/tests/*.test.js --recursive ./server/tests/**/*.test.js --exit",
     "coveralls": "nyc report --reporter=text-lcov | coveralls",
-    "start": "npm run migrationSeeds && babel-node -- server/index.js",
-    "start:dev": "nodemon --exec babel-node ./server/index.js",
+    "start": "npm run migration && npm run migration:seed && babel-node -- server/index.js",
+    "start:dev": "npm run migration:reset && nodemon --exec babel-node ./server/index.js",
```

Comment:
I think the `start:dev` can remain as it was because this **npm run migration:reset** would not work for windows users, as you remember.

Reply:
@irfiacre, I added separate scripts for Windows machines... You will see them in the `package.json`... I think just use these instead.

```
"win:start:dev": "nodemon --exec babel-node ./server/index.js",
"win:createTables": "npx sequelize-cli db:migrate",
"win:drop": "npx sequelize-cli db:migrate:undo:all",
"win:devtable": " npm run drop && npm run createTables && npx sequelize-cli db:seed:all"
```

Reply:
but since `start:dev` was working for all of us and I guess on `start:dev` migrations are not that necessary, I think It's better to keep one script!

Reply:
I think it would be better to have both since I'm running migrations on my machine when using `start:dev` script (It makes it easier for me since I don't have to run migrations separately, and I just do it all in one script)...

Maybe we can first hear what @bdushimi and @Francois-MUGOROZI have to say about it.

Reply:
I understand now.

Reply:
@irfiacre The other option would be to leave `start:dev` as it was originally, and then instead create another script: ` "unix:start:dev": "npm run migration:reset && nodemon --exec babel-node ./server/index.js" ` which would be used by those with Unix machines.

So I believe we would have to agree as a team which options would be better...